cli/server/previewServer.js

```diff
@@ -30,6 +30,26 @@ const PUBLIC_CONFIG_BASENAME = 'cache-public.config';
 const PUBLIC_CONFIG_PATH = path.join(CACHE_DIRECTORY, PUBLIC_CONFIG_BASENAME);
 const FAV_ICON_PATH = './node_modules/jest-preview/cli/server/favicon.ico';
 
+const updateNotifier = require('update-notifier');
+const chalk = require('chalk'); // Already installed in `update-notifier`
+
+// Checks for available update
+const notifier = updateNotifier({
+  pkg: require('../../package.json'),
+  updateCheckInterval: 0, // How often to check for updates
+  shouldNotifyInNpmScript: true, // Allows notification to be shown when running as an npm script
+  distTag: 'latest' // Can be use to notify user about pre-relase version
+})
+
+notifier.notify({
+  defer: true, // Try not to annoy user by showing the notification after the process has exited
+  message:[
+      `${chalk.blue('{packageName}')} has an update available: ${chalk.gray('{currentVersion}')} → ${chalk.green('{latestVersion}')}`,
+      `Please run ${chalk.cyan('`{updateCommand}`')} to update.`,
+      chalk.dim(`If you don\'t want to receive this notification, please set \`optOut\` property in \n\`${notifier.config.path}\` to true\nor running jest-preview with \`--no-update-notifier\` flag.`),
```

Comment:
I think this is not needed, since we always want to encourage them to update to the newest version. Plus, it's only a friendly reminder, what is the bad DX it can bring?

Reply:
This adds noise to the developer experience as well, let's just keep the message short and concise.
<img width="1274" alt="notifier new version" src="https://user-images.githubusercontent.com/8603085/170899100-ef3dcfa5-0b2f-4a00-835a-8faa502bb9b7.png">

Reply:
@nvh95 Hmm, I guess I can remove this dimmed text and then mention it in somewhere in the document (like in the Others` section?

Reply:
@huynhducduy I don't think it's needed. Always let our users know there is a new `latest` version available. I don't think our users will get mad since we persist this message (especially we have `defer` mode, they just exit the app and really don't care)

Reply:
@huynhducduy Just a friendly suggestion, it would be nice if you can push the change then I mark the feedback/ comment as `resolved` later. It's easier for reviewers to track your updates.
My preferred option is not to resolve if there are not many discussions 😂.